packages/common/src/util/base64.js

```diff
@@ -22,5 +30,13 @@ export const encode = data => Buffer.from(data, 'utf-8').toString('base64');
  * const decoded = decode('SGVsbG8gV29ybGQ=');
  * console.log(decoded); // Output: Hello World
  */
-export const decode = base64Data =>
-  Buffer.from(base64Data, 'base64').toString('utf-8');
+export const decode = base64Data =>{
+  let decodedObject = Buffer.from(base64Data, 'base64').toString('utf-8');
+
+  if(['[', '{'].includes(decodedObject[0])){
```

Comment:
Hmm, this doesn't seem so robust. I think `startsWith('[') || startsWith('{') ` is safer? Or you could write it in regex to make it a bit cleaner.

As it stands, if the decoded string includes a square bracket or brace anywhere, we'll try to JSON parse it, which is likely to cause errors